src/main/resources/database/changes/release-10.49.0/add_case_group_status_field.sql

```diff
@@ -0,0 +1,2 @@
+ALTER TABLE casesvc."casegroup"
+    ADD COLUMN casegroupstatus character varying(20);
```

Comment:
This needs to change to just 'status'

Reply:
Need to update API.md to show the status in the CaseGroup record.

Reply:
We need to add a CaseGroupStatusAudit to record state transitions, PartyID and date/time.